package.json

```diff
@@ -6,10 +6,7 @@
     "backstop": "./cli/index.js"
   },
   "scripts": {
-    "lint": "npm run lint-web -- --fix && npm run lint-node -- --fix",
-    "lint-ci": "npm run lint-web && npm run lint-node",
-    "lint-web": "eslint --env browser compare/*.js compare/src/**/*.js",
-    "lint-node": "eslint --env node core/*.js core/**/*.js cli/*.js cli/**/*.js capture/*.js capture/**/*.js",
+    "lint": "eslint '*.js' 'compare/*.js' 'compare/src/**/*.js' 'core/**/*.js' 'cli/**/*.js' 'capture/**/*.js'",
```

Comment:
Combining all these scripts into one since what env to use is controlled by `.eslintrc` files.

Reply:
After this is merged we should probably enabled linting for test directory too.